src/ReadLoader.cpp

```diff
@@ -663,7 +669,7 @@ void ReadLoader::processCompressedRunnieFile()
     // Use multithreaded code to store the reads.
     setupLoadBalancing(readIdTable.size(), 1000);
     runThreads(&ReadLoader::processCompressedRunnieFileThreadFunction, threadCount);
-    compressedRunnieReader = 0;
+    // compressedRunnieReader = 0;
```

Comment:
We still want the equivalent of this for `unique_ptr` (`reset`?), to make sure the reader gets destroyed.

Reply:
It gets destroyed when it goes out of scope, few lines below this, where the method ends.

Reply:
It is a data member, so it does not go out of scope at the end of this function.

Reply:
You're right. This is a class data member.